src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === activeTab.id ? 'is-active' : ''}
+              data-cy="Tab"
+              onClick={() => {
+                if (tab.id !== activeTab.id) {
+                  onTabSelected(tab.id);
+                }
+              }}
+              onKeyDown={event => {
+                if (event.key === 'Enter' || event.key === ' ') {
+                  if (tab.id !== activeTab.id) {
+                    onTabSelected(tab.id);
+                  }
+                }
+              }}
```

Comment:
you don't need this. and it is not working)

Reply:
When I remove this event handler, I get a linter error - Visible, non-interactive elements with click handlers must have at least one keyboard listener.